File: app/src/main/java/com/neome/api/meta/base/dto/EntVdImageTransforms.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdVdImageFunc;

import org.jetbrains.annotations.Nullable;

import java.util.Map;

@SuppressWarnings("unused")
public class EntVdImageTransforms extends EntVdAutoStep
{
  @Nullable
  public StudioDtoArgValueParameter inputField;

  public MetaIdVdImageFunc[] keys;

  public Map<MetaIdVdImageFunc, ImageXform> map;

  @Nullable
  public StudioDtoArgValueParameter outputField;
}

File: app/src/main/java/com/neome/api/meta/base/dto/EntVdImageTransforms.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.EntVdAutoStep
import com.neome.api.meta.base.dto.ImageXform
import com.neome.api.meta.base.Types.MetaIdVdImageFunc
import com.neome.api.meta.base.dto.StudioDtoArgValueParameter

interface EntVdImageTransforms : EntVdAutoStep
{
  val inputField: StudioDtoArgValueParameter?
  val keys: List<MetaIdVdImageFunc>
  val map: Map<MetaIdVdImageFunc, ImageXform>
  val outputField: StudioDtoArgValueParameter?
}